paddle/fluid/framework/operator.cc

```diff
@@ -1061,8 +1061,8 @@ proto::VarType::Type OperatorWithKernel::IndicateDataType(
           t = &(var->Get<SelectedRows>().value());
         }
         if (t != nullptr) {
-          PADDLE_ENFORCE(t->IsInitialized(), "Input %s is not initialized: %s",
-                         ipt_name, DebugString());
+          PADDLE_ENFORCE(t->IsInitialized(), "Input %s is not initialized",
```

Comment:
Is it suitable to delete `DebugString` directly? Or we distinguish `Release mode` and `Debug mode`?
@panyx0718 @Superjomn How do you think about it?

Reply:
I think this change is ok.

Reply:
Discussed with @qingqing01, we think this PR is OK.